versions.props

```diff
@@ -69,6 +69,7 @@ org.slf4j:* = 1.7.32
 org.wildfly.common:wildfly-common = 1.5.4.Final
 org.xerial:sqlite-jdbc = 3.30.1
 org.yaml:snakeyaml = 1.29
+org.openjdk.jmh:* = 1.33
```

Comment:
nit: can we lexicographically sort this to line 67?

Reply:
👍